sdk/src/wallet/account/operations/participation/voting_power.rs

```diff
@@ -58,10 +58,10 @@ where
             Some(current_output_data) => {
                 let output = current_output_data.output.as_basic();
 
-                // TODO checked addition
+                let new_amount = output.amount().checked_add(amount).ok_or(Error::InvalidVotingPower)?;
```

Comment:
Maybe more descriptive? like how much is missing or what we tried to add

Reply:
Not much we can do, it can overflow so we'd have to store the addition in an u128, and for the subtraction it gets even weirder

Reply:
And if we want to only store what we tried to add/sub, we'd need two variants? Otherwise it's confusing

Reply:
I also think without value it's fine, if the code is not wrong, it shouldn't return an error at all